### DISCUSSION TITLE
Why does the PostCSSModules plugin force the CSS to be embedded in JS Files?

### DISCUSSION THREAD

**Discussion by gyfchong** [glromeo/esbuild-sass-plugin#123] [Q&A]
I'm trying to create a single CSS file for consumers of my design system components to import and utilise in SSR applications. Existing bundling in consumer apps create a CSS file, and I'd like to replicate this format.

However this `postcssModules` has the following limitations:
> postcssModules produces Javascript modules which are handled by esbuild's js loader, so the type option is ignored

Is there a way around this? Happy to make a PR with the right direction.

**Comment by glromeo** [comment]
Hey there... sorry for the (very) late reply
I think your question has been addressed
so I close tis Q&A... apologies for not having
seen it ...or did I? 🤔
